lib/helpbase.gi

```diff
@@ -1013,13 +1013,74 @@ InstallGlobalFunction(HELP_GET_MATCHES, function( books, topic, frombegin )
   return [exact, match];
 end);
 
+if not IsBound( InitialSubstringUTF8String ) then
+  InitialSubstringUTF8String:= "dummy";
+fi;
+if not IsBound( LETTERS ) then
+  LETTERS:= "dummy";
+fi;
+if not IsBound( WidthUTF8String ) then
+  WidthUTF8String:= "dummy";
+fi;
```

Comment:
I assume all these globals come from GAPDoc, and this code is here to ensure this file loads (without syntax warnings) regardless of whether GAPDoc is loaded first or not? Perhaps a code comment could be added to clarify this?